views/assets/javascripts/tyk.js

```diff
@@ -11,3 +11,13 @@ function onCancel() {
 	}
 }
 
+document.getElementById("profile-icon").addEventListener("mouseover", mouseOver);
+document.getElementById("logout-btn").addEventListener("mouseout", mouseOut);
```

Comment:
Should these two events be attached to different elements? It seems weird.

Reply:
Can you check again please, just modified it slightly.
The idea is to show the "dropdown" menu when hovering over the icon but hide it when hovering elsewhere

Reply:
@ConsM you can do this only from css, no js needed. 
```
.profile-icon:hover + profile-menu {
    display: block;
}
```

Reply:
@lghiur with this approach when the cursor moves out of the icon the menu disappears instantly , its impossible to hit the logout btn

Reply:
@ConsM can we try smth like this? https://www.w3schools.com/css/css_dropdowns.asp

Reply:
if you make icon wrapper sticked to the dropdown one, it shouldn't dissapear

Reply:
https://www.w3schools.com/css/tryit.asp?filename=trycss_dropdown_text